mercury_app/utils.py

```diff
@@ -514,10 +514,23 @@ def get_json_donut(percentage, name, id_int):
 
 def get_summary_types_handed(order_ids):
     total_mercha = Merchandise.objects.filter(order_id__in=order_ids).values(
-        'name').annotate(name_count=Count('name')).order_by('name')
-    handed_mercha = Merchandise.objects.filter(
-        order_id__in=order_ids, delivered=True).values(
-        'name').annotate(name_count=Count('name')).order_by('name')
+             'name').annotate(name_count=Sum('quantity')).order_by('name')
+    types = Merchandise.objects.filter(order_id__in=order_ids).values_list('name', flat=True).distinct()
+    types_id = []
+    for ty_pe in types:
+        types_id.append(Merchandise.objects.filter(order_id__in=order_ids, name=ty_pe))
+    ids_mercha = Merchandise.objects.filter(order_id__in=order_ids)
+    type_transaction_count = []
+
+    for ty_pe in types_id:
```

Comment:
Do you really need to name it 'ty_pe'? is some sort of  internal convention? it would be great if you can use the best description you have